### DISCUSSION TITLE
постоянная перезагрузка ESP32 Wroom-32 30pin

### DISCUSSION THREAD

**Discussion by uriy1000** [vvip-68/LedPanelWiFi#84] [Вопросы и ответы] [answered]
вот что творится на сом порту
2019 12:21:46
13:34:43.432 -> 
13:34:43.432 -> rst:0x3 (SW_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
13:34:43.432 -> configsip: 0, SPIWP:0xee
13:34:43.432 -> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
13:34:43.432 -> mode:DIO, clock div:1
13:34:43.432 -> load:0x3fff0030,len:1344
13:34:43.480 -> load:0x40078000,len:13964
13:34:43.480 -> load:0x40080400,len:3600
13:34:43.480 -> entry 0x400805f0
13:34:43.480 -> ets Jul 29 2019 12:21:46
13:34:43.480 -> 
13:34:43.480 -> rst:0x3 (SW_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
13:34:43.480 -> configsip: 0, SPIWP:0xee
13:34:43.480 -> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
13:34:43.480 -> mode:DIO, clock div:1
13:34:43.480 -> load:0x3fff0030,len:1344
13:34:43.480 -> load:0x40078000,len:13964
13:34:43.480 -> load:0x40080400,len:3600
13:34:43.480 -> entry 0x400805f0
13:34:43.480 -> ets Jul 29 2019 12:21:46
13:34:43.480 ->

**Comment by vvip-68** [comment]
К плате что-нибудь подключено?
Какие параметры компиляции в IDE? Какая именно плата там выбрана?

**Comment by uriy1000** [comment, marked as answer]
к плате не чего не подключено. плата вот такая  ----------------23.11.2024, 13:52, "vvip-68" ***@***.***>:Кому: vvip-68/LedPanelWiFi ***@***.***);Копия: Author ***@***.***);Тема: [vvip-68/LedPanelWiFi] постоянная перезагрузка ESP32 Wroom-32 30pin (Discussion #84);  К плате что-нибудь подключено?Какие параметры компиляции в IDE? Какая именно плата там выбрана?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>    -- С уважением,Юрий ***@***.***

**Comment by uriy1000** [comment]
Технические характеристики:
Модель макетной платы: ESP32-DevKitC-32 30P
Модель модуля: ESP32-WROOM-32
Основной управляющий чип: ESP32-DOWDQ5-V3, двухъядерный 32-битный MCU, интегрированный WiFi, Bluetooth.
Интегрированная 520-kBSRAM, 448-kBROM16-kBSRAMinRTC
Внешнее хранилище: 4 МБ Чип
USB-драйвера: CH340C с более высокой скоростью загрузки и большей стабильностью.
Поддержка внешнего широкого входа напряжения VIN 5-12 В (версия батареи максимум на вход 5,5 В)
Поддержка источника питания USB, внешнего источника питания 3,3 В и источника питания VIN три вида источника питания
Поддержка ArduinoIDE mixly, Mind +, Python и другого программного обеспечения

**Reply by vvip-68** [reply]
А размер матрицы какой задан? Может много задали и памяти не хватает?

**Comment by uriy1000** [comment]
17 на 11----------------
                        23.11.2024, 21:43, "vvip-68" ***@***.***>:
                        Кому: vvip-68/LedPanelWiFi ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [vvip-68/LedPanelWiFi] постоянная перезагрузка ESP32 Wroom-32 30pin (Discussion #84);
                        
                        
А размер матрицы какой задан? Может много задали и памяти не хватает?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

                        -- С уважением,Юрий ***@***.***

**Comment by uriy1000** [comment]
какую плату  посоветуете----------------
                        23.11.2024, 21:43, "vvip-68" ***@***.***>:
                        Кому: vvip-68/LedPanelWiFi ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [vvip-68/LedPanelWiFi] постоянная перезагрузка ESP32 Wroom-32 30pin (Discussion #84);
                        
                        
А размер матрицы какой задан? Может много задали и памяти не хватает?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

                        -- С уважением,Юрий ***@***.***

**Reply by vvip-68** [reply]
17x11 - совсем не много. Даже ESP8266 легко потянет.
Этот вариант ESP32, что у вас - пожалуй самый лучший из ESP32. 

А у вас этот размер точно задан? А то вон у Гайвера на форуме кто-то указал, что у него матрица 4000x4000 - то есть 16 миллионов диодов. Хотя, думаю, на самом деле физическая матрица примерно 20x20. Зачем он это сделал - не знаю.

С такой проблемой что у вас - не сталкивался ни разу. Если все настройки правильно указаны.
А если как в том случае - 16 млн диодов - то и не удивительно что не запускается.

**Comment by uriy1000** [comment]
добрый деньESP32 s3 после загрузки в мониторинг порта не чего не происходит ----------------
                        Кому: vvip-68/LedPanelWiFi ***@***.***), vvip-68/LedPanelWiFi ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [vvip-68/LedPanelWiFi] постоянная перезагрузка ESP32 Wroom-32 30pin (Discussion #84);
                        23.11.2024, 22:12, "Попов Юрий" ***@***.***>:
                        какую плату  посоветуете----------------
                        23.11.2024, 21:43, "vvip-68" ***@***.***>:
                        Кому: vvip-68/LedPanelWiFi ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [vvip-68/LedPanelWiFi] постоянная перезагрузка ESP32 Wroom-32 30pin (Discussion #84);
                        
                        
А размер матрицы какой задан? Может много задали и памяти не хватает?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

                        -- С уважением,Юрий ***@***.***
-- С уважением,Юрий ***@***.***

**Comment by vvip-68** [comment]
Тогда у вас вероятно неисправный микроконтроллер.
Нет таких ситуаций, когда бы исправный контроллер ничего не выводил в монитор порта. 
Он начинает выводить туда диагностику еще до того, как запускается собственно сам скетч гирлянды.
Если вывода нет - это или неисправный микроконтроллер или он загнан подключением пинов в режим, когда не стартует, а вводится в ожидание загрузки скетча. Или вы открываете не тот компорт, который, собственно, формирует микроконтроллер.

**Comment by uriy1000** [comment]
как узнать что он загнан подключением пинов----------------
                        Кому: vvip-68/LedPanelWiFi ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [vvip-68/LedPanelWiFi] постоянная перезагрузка ESP32 Wroom-32 30pin (Discussion #84);
                        15.12.2024, 21:09, "vvip-68" ***@***.***>:
                        
Тогда у вас вероятно неисправный микроконтроллер.
Нет таких ситуаций, когда бы исправный контроллер ничего не выводил в монитор порта.
Он начинает выводить туда диагностику еще до того, как запускается собственно сам скетч гирлянды.
Если вывода нет - это или неисправный микроконтроллер или он загнан подключением пинов в режим, когда не стартует, а вводится в ожидание загрузки скетча. Или вы открываете не тот компорт, который, собственно, формирует микроконтроллер.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
-- С уважением,Юрий ***@***.***

**Reply by vvip-68** [reply]
Прочитать в документации на микроконтроллер какие пины опрашиваются при его старте, какие уровни должны быть на этих пинах в момент запуска для перевода его либо в рабочий режим, либо в один из режимов загрузки.

Проще отпаять все что припаяно и проверить, что он стартует без подключенных к пинам устройств, которые могут менять уровни на управляющих пинах в момент запуска микроконтроллера.

Если не запускается на голом микроконтроллере и он при старте даже не выводит своей диагностической информации) - значит либо сгорел, либо вы не подключились к тому ком-порту через который идет вывод информации из контроллера вовне.

**Comment by uriy1000** [comment]
он голый а к какому порту при загрузки все принимает пишет перезагрузка и все----------------
                        Кому: vvip-68/LedPanelWiFi ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [vvip-68/LedPanelWiFi] постоянная перезагрузка ESP32 Wroom-32 30pin (Discussion #84);
                        16.12.2024, 06:46, "vvip-68" ***@***.***>:
                        
Прочитать в документации на микроконтроллер какие пины опрашиваются при его старте, какие уровни должны быть на этих пинах в момент запуска для перевода его либо в рабочий режим, либо в один из режимов загрузки.
Проще отпаять все что припаяно и проверить, что он стартует без подключенных к пинам устройств, которые могут менять уровни на управляющих пинах в момент запуска микроконтроллера.
Если не запускается на голом микроконтроллере и он при старте даже не выводит своей диагностической информации) - значит либо сгорел, либо вы не подключились к ком-порту через который идет вывод информации из контроллера вовне.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
-- С уважением,Юрий ***@***.***
